manifest/resolver.go

```diff
@@ -797,3 +832,54 @@ func mustAbs(action Action, path string) error {
 	}
 	return participle.Errorf(action.position(), "%q must be an absolute path", path)
 }
+
+// resolveJSONVariable fetches JSON data and extracts a value using JSONPath.
+func resolveJSONVariable(client *http.Client, jsonConfig *JSONAutoVersionBlock, path string) (string, error) {
```

Comment:
The auto-versioning part makes sense, but issuing HTTP requests during package resolving is going to result in Hermit being really slow. I do understand that this makes it impossible to auto-version some packages, but we'll have to come up with a different approach. Perhaps auto-versioning can pre-populate vars that then get propagated somehow to version-specific vars (this doesn't exist yet).

Reply:
Ah fair point, I missed that nuance. Lemme go back to the drawing board on this bit.